interceptor/go.mod

```diff
@@ -0,0 +1,10 @@
+module interceptor
```

Comment:
I would put these in webrtc/pkg/interceptor for now.

It would be interesting to move them out of tree eventually. I just don't see a good way to do it yet.

When we do it will be easy enough to alias though!